src/tree/GitHubConfigGroupTreeItem.ts

```diff
@@ -6,11 +6,14 @@
 import { pathExists, readdir } from "fs-extra";
 import { basename, join } from "path";
 import { ThemeIcon } from "vscode";
-import { AzExtParentTreeItem, AzExtTreeItem, IActionContext, TreeItemIconPath } from "vscode-azureextensionui";
+import { AzExtParentTreeItem, AzExtTreeItem, IActionContext, InvalidTreeItem, TreeItemIconPath } from "vscode-azureextensionui";
+// eslint-disable-next-line import/no-internal-modules
+import { YAMLSyntaxError } from "yaml/util";
```

Comment:
No other way to import? 😬

Reply:
None that I'm aware of ☹️.  We added the `import/no-internal-modules` rule recently when we switched to the shared eslint config.